src/MergeContext.js

```diff
@@ -840,7 +842,7 @@ class PullRequest {
     staged() { return this._prState.staged(); }
 
     _debugString() {
-        const staged = this._stagingSha ? "staged: " + this._stagingSha.substr(0, this._shaLimit) + ' ' : "";
+        const staged = this._tagSha ? "staged: " + this._tagSha.substr(0, this._shaLimit) + ' ' : "";
```

Comment:
Should the `staged` string be limited to _staged_ PRs?

* If yes:
```suggestion
        const staged = this.staged() ? "staged: " + this._tagSha.substr(0, this._shaLimit) + ' ' : "";
```
* Otherwise:
```suggestion
        const tagged = this._tagSha ? "tagged: " + this._tagSha.substr(0, this._shaLimit) + ' ' : "";
```

Reply:
> Should the `staged` string be limited to _staged_ PRs?

I would print this tag sha in any case, which may be useful. I renamed to 'tagged' as you suggested.